manifests/extension.pp

```diff
@@ -142,74 +142,76 @@
     $package_depends = undef
   }
 
-  if $zend == true {
-    $extension_key = 'zend_extension'
-    if $php_api_version != undef {
-      $module_path = "/usr/lib/php5/${php_api_version}/"
+  if $provider != 'pear' { # PEAR packages don't require antu further configuration, they just need to "be there".
+    if $zend == true {
+      $extension_key = 'zend_extension'
+      if $php_api_version != undef {
+        $module_path = "/usr/lib/php5/${php_api_version}/"
```

Comment:
basicly, it wouldn't have worked before... I guess...
I'll look into this, but maybe we should do it in a seperate pull-req since it exceeds the scope of this one?